util/Migrator/DbScripts/2021-07-07_00_FixBulkConfirm.sql

```diff
@@ -0,0 +1,26 @@
+IF OBJECT_ID('[dbo].[User_ReadPublicKeysByOrganizationUserIds]') IS NOT NULL
+    BEGIN
+        DROP PROCEDURE [dbo].[User_ReadPublicKeysByOrganizationUserIds]
+    END
+GO
+
+CREATE PROCEDURE [dbo].[User_ReadPublicKeysByOrganizationUserIds]
+    @OrganizationId UNIQUEIDENTIFIER,
+    @OrganizationUserIds [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        OU.[Id],
+        OU.[UserId],
+        U.[PublicKey]
+    FROM
+        @OrganizationUserIds OUIDs
+            INNER JOIN
```

Comment:
Indention looks funny. (Also applies to line 22, 25)

Reply:
Fixed in both files